i18n/en.json

```diff
@@ -15,7 +15,10 @@
 	"wikibasereconcileedit-editendpoint-qualifiers-references-not-supported": "Qualifiers and References are not currently supported",
 	"wikibasereconcileedit-editendpoint-reconciliation-property-missing-in-statements": "Entity must have at least one statement for the reconciliation Property",
 	"wikibasereconcileedit-editendpoint-invalid-reconciliation-statement-type": "Reconciliation statement must contain a value",
+	"wikibasereconcileedit-editendpoint-invalid-request-body": "Invalid or empty JSON request body",
+	"wikibasereconcileedit-editendpoint-invalid-content-type": "Content-type should be application/json",
 	"wikibasereconcileedit-reconciliationservice-matched-multiple-items": "The reconciliation is ambiguous; matched {{PLURAL:$2|item|$2 items}}: $1",
 	"wikibasereconcileedit-minimaliteminput-required-keys": "Statements missing '$1' key",
-	"wikibasereconcileedit-fullwikibaseiteminput-unsupported-type": "The only supported entity {{PLURAL:$2|type is|types are}} $1."
+	"wikibasereconcileedit-fullwikibaseiteminput-unsupported-type": "The only supported entity {{PLURAL:$2|type is|types are}} $1.",
+	"wikibasereconcileedit-unauthorized-access": "Not logged in."
```

Comment:
Is it intentional that this doesn’t mention the possibility of an invalid CSRF token?

Reply:
I updated to be a bit more specific what could be wrong.